### DISCUSSION TITLE
Cycling aging

### DISCUSSION THREAD

**Discussion by epsilon-smaller-zero** [pybamm-team/PyBaMM#3102] [Q&A] [answered]
Hello everyone,

I am quite new to PyBaMM and i have a question regarding cycling aging. I wanted to perform a cycling aging with the following experiment:

1. Loop:
- Charge until 4.2V with 0.2C
- Rest for 30 minutes
- Discharge until 60% (3.72V) with 0.2C
- Rest for 30 min
- Discharge for 30 seconds with 0.2C
- Rest for 30 min
- Discharge until 3V with 0.2C

2. Loop:
- Charge until 4.2V with 1C
- Discharge until 3 V with 1C

The cycling should look like (Loop 1, Loop2 *24)*20.

I used this  [example](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/cycling_ageing.py) as a basis and I have just tried to change the experiment parameters. 
The problem is the second loop. It is marked yellow with the error message: 
"Expected type "list[tuple[str,str,str,str,str,str,str,str]]" (matched generic type "list[_T]"), got "list[tuple[str,str,str]]" instead."
What is the problem here and how can I solve it?
Secondly, how can I write the code, that the 2. loop is taken 24 times and the whole procedure 20 times?

My code:

```python3
pb.set_logging_level("NOTICE")
model = pb.lithium_ion.DFN(
    {
        "SEI": "ec reaction limited",
        "SEI film resistance": "distributed",
        "SEI porosity change": "true",
        "lithium plating porosity change": "true",
    }
)

param = pb.ParameterValues("Chen2020")

experiment = pb.Experiment(
    [
        (
            "Charge at 0.2 C until 4.2 V",
            "Rest for 30 minutes",
            "Discharge at 0.2 C until 3.72 V",
            "Rest for 30 minutes",
            "Discharge at 0.2 C for 30 seconds",
            "Rest for 30 minutes",
            "Discharge at 0.2 C at 3V",
        )
    ]
    + [
        (
            "Charge at 1C until 4.2 V",
            "Discharge at 1C until 3 V",
        )
    ]*24
)

sim = pb.Simulation(model, experiment=experiment, parameter_values=param)
sim.solve(solver=pb.CasadiSolver(mode="fast with events"))
```

Thank you for your help in advance!

**Comment by brosaplanella** [comment, marked as answer]
I got a different error, caused by a typo: it should say `"Discharge at 0.2 C until 3V"` instead of `"Discharge at 0.2 C at 3V"`. Fixing that got the code running at my end.

**Reply by epsilon-smaller-zero** [reply]
Thank you for your help. It works now :)
